cmd/query/app/qualitymetrics/data.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package qualitymetrics
+
+func GetSampleData() TQualityMetrics {
+	return TQualityMetrics{
+		TraceQualityDocumentationLink: "https://www.jaegertracing.io/docs/1.18/",
```

Comment:
again, why this random link? If we have _some_ docs about this feature, e.g. a README in some repo (maybe in jaeger-analytics) then link there. If there isn't any, then create a minimal / high-level README in this package and point to it.

Reply:
https://github.com/jaegertracing/jaeger-analytics-java#trace-quality-metrics

Reply:
https://github.com/jaegertracing/jaeger/issues/367